src/components/updated-terms-conds/updated-terms-conds-controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from "express";
+import {PATH_NAMES} from "../../app.constants";
+import {ExpressRouteFunc} from "../../types";
+import {updateTermsAndCondsService} from "./updated-terms-conds-service";
+import {UpdateTermsAndCondsServiceInterface} from "./types";
+
+export function updatedTermsCondsGet(
+    service: UpdateTermsAndCondsServiceInterface = updateTermsAndCondsService()
+): ExpressRouteFunc {
+  return async function (req: Request, res: Response) {
+    const sessionId = res.locals.sessionId;
+    const clientSessionId = res.locals.clientSessionId;
+    const clientInfo = await service.clientInfo(sessionId, clientSessionId);
+    req.session.redirectUri = clientInfo.redirectUri;
+
+    res.render("updated-terms-conds/index.njk");
+  };
+}
+
+export function updatedTermsCondsPost(
+    service: UpdateTermsAndCondsServiceInterface = updateTermsAndCondsService()
+): ExpressRouteFunc {
+  return async function (req: Request, res: Response) {
+    const updatedTermsAndCondsValue = true;
+    const {email} = req.session.user;
+    const sessionId = res.locals.sessionId;
+    const redirectUri = req.session.redirectUri;
+    const acceptOrReject = req.body.acceptOrReject;
+
+    if (acceptOrReject === "reject") {
+      res.redirect(redirectUri);
```

Comment:
i think we want to redirect to the service with an error message so would be something like:
`res.redirect(redirectUri+ "?error_code=rejectedTermsAndConditions");`